clients/src/main/java/org/apache/kafka/common/requests/ReadShareGroupStateResponse.java

```diff
@@ -65,4 +68,50 @@ public static ReadShareGroupStateResponse parse(ByteBuffer buffer, short version
         new ReadShareGroupStateResponseData(new ByteBufferAccessor(buffer), version)
     );
   }
+
+  public static ReadShareGroupStateResponseData getSuccessResponse(
```

Comment:
These aren't really getters. I suggest "toResponseData" as a method name. I think that pattern is used in other places in Kafka such as FetchResponse.